crates/biome_html_parser/src/lexer/tests.rs

```diff
@@ -150,17 +153,11 @@ fn element() {
 }
 
 #[test]
-fn element_with_text() {
```

Comment:
Was this test changed for some reason?

Reply:
This test was removed because lexing it correctly requires changing contexts, which is not supported by the `assert_lex!` macro.